src/ontology/components/mondo-tags.owl

```diff
@@ -38684,5 +38682,5 @@
 
 
 
-<!-- Generated by the OWL API (version 4.5.6) https://github.com/owlcs/owlapi -->
+<!-- Generated by the OWL API (version 4.5.25.2023-02-15T19:15:49Z) https://github.com/owlcs/owlapi -->
```

Comment:
@sabrinatoro I'm not sure why these changes were added. Do you know if this is okay?

Reply:
I think that it is related to the fact that I removed a "exclude from QC" annotation, and it automatically updated the "tags" component. So I "think" it is ok. But maybe @matentzn can confirm?

Reply:
Absolutely this is the correct behaviour!

Reply:
good to know, thank you!